openedx/features/job_board/views.py

```diff
@@ -15,6 +16,21 @@ class JobListView(ListView):
     template_engine = 'mako'
 
 
-def show_job_detail(request):
-
-    return render_to_response('features/job_board/job_detail.html', {})
+def show_job_detail(request, job_id):
+    job = get_object_or_404(
```

Comment:
@ahmed-faraz46 do we have a 404 page in our app where we can redirect it to in case job doesn't exists?